grakn-core/src/main/java/ai/grakn/GraknGraph.java

```diff
@@ -64,8 +63,8 @@
      * @param label A unique label for the {@link EntityType}
      * @return A new or existing {@link EntityType} with the provided label
      *
-     * @throws GraphRuntimeException if the graph is closed
-     * @throws ConceptNotUniqueException if the {@param label} is already in use by an existing non-{@link EntityType}.
+     * @throws GraphOperationException if the graph is closed
+     * @throws PropertyNotUniqueException if the {@param label} is already in use by an existing non-{@link EntityType}.
```

Comment:
Maybe this is for a future PR, but this should say `throws` on the method signature as well.
Ditto for a lot of methods in this PR.